amplitude.js

```diff
@@ -439,6 +444,30 @@ Amplitude.prototype.logEvent = function(eventType, eventProperties) {
   }
 };
 
+Amplitude.prototype.logEvent = function(eventType, eventProperties) {
+  this._logEvent(eventType, eventProperties);
+};
+
+// Test that n is a number or a numeric value.
+var _isNumber = function(n) {
+  return !isNaN(parseFloat(n)) && isFinite(n);
+};
+
+Amplitude.prototype.logRevenue = function(price, quantity, product) {
+  // Test that the parameters are of the right type.
+  if (!_isNumber(price) || quantity !== undefined && !_isNumber(quantity)) {
```

Comment:
oh i see here that i think it is useful to note that the every param 's data type on the readme file of amplitude-js_2.x;